devenv/docker/blocks/tempo/docker-compose.yaml

```diff
@@ -1,8 +1,111 @@
-  tempo:
-    image: grafana/tempo:latest
-    command: ["-config.file=/etc/tempo.yaml"]
+x-logging: &default-logging
```

Comment:
Want to keep or remove this logger? I think it really slows down bringing up and down the containers so I have it commented out locally. 

I think the load generator already generates logs so I don't think this is needed for testing.

Reply:
There does not seem to be any logs without it!

Reply:
Good call then, I thought the load generator created some, but in that case lets leave it in